main.js

```diff
@@ -179,6 +179,18 @@ class Sourceanalytix extends utils.Adapter {
             }
         }
 
+        // Load time definitions
+        catArray = ['Hour'];
+        for (const item in catArray) {
+            const unitItem = adapterHelpers.units.time[catArray[item]];
```

Comment:
having the section loaded on .time limits us to only calculate hour. In current why of unit transition I cannot handle hour conversation (like s => m => h) as the exponents are missing.,

will check with helper lib adding the missing values and than we can use all time units including conversion

Reply:
We can not use exponents here I think. Aren't they of base 10? Time calculations are 60 and 24. Perhaps a 'factor' with the real number would be more sufficient as a base 10 exp

Reply:
good one, naming it exponents would incorrect its indeed a factor calculation. the factor should be set in the library we will only consume it basically in SA

Reply:
A really crappy hack would be log(60) and log(24). That would end in a correct base 10 exponent. But we will get problems with incorrect calculations due to rounding issues 🥳